src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.windows.cs

```diff
@@ -341,9 +341,11 @@ void OnMediaElementMediaOpened(WindowsMediaElement sender, object args)
 		{
 			SetDuration(MediaElement, Player);
 		}
-		MediaElement.MediaOpened();
-
-		static void SetDuration(in IMediaElement mediaElement, in MediaPlayerElement mediaPlayerElement) => mediaElement.Duration = mediaPlayerElement.MediaPlayer.NaturalDuration == TimeSpan.MaxValue
+		if (MediaElement.Duration > TimeSpan.Zero)
+		{
```

Comment:
should this change be here? This PR is to fix just the Android issue

Reply:
Reverted all changes to windows. Missed that. TY good catch. Fix should just be for android resources now.